src/raw/dockerapi/fetcher.go

```diff
@@ -69,3 +90,41 @@ func (f *Fetcher) ContainerStats(ctx context.Context, containerID string) (types
 	}
 	return statsJSON, nil
 }
+
+// getUint64FromMapAndLog retrieves a uint64 value from a map based on the key, logs an error if the key is not found.
+func getUint64FromMapAndLog(m map[string]uint64, key string) uint64 {
+	if val, ok := m[key]; ok {
+		return val
+	}
+	log.Error("key %s not found in map", key)
```

Comment:
Should we log an error? I wonder if a warning would be enough. I also wonder if it would be possible to add more information to the message. For instance, if ever encountered, `key file not found in map` wouldn't be very helpful, would it?

Reply:
Agree, a warning might be better. However, what else would you add? I added the key since it's useful to see if what we are looking for... but the other part I see is to log the full stats map 🤔